projects/adrv9009zu11eg/common/som_connector_2.txt

```diff
@@ -0,0 +1,70 @@
+A14       GPIO_12_B             AV19        IO_L6N_T0U_N11_AD6N_64      adrv9009_gpio_12_b     LVCMOS18     #N/A
+A15       GPIO_13_B             AM18        IO_L7N_T1L_N1_QBC_AD13N_64  adrv9009_gpio_13_b     LVCMOS18     #N/A
+A16       GPIO_14_B             AM19        IO_L7P_T1L_N0_QBC_AD13P_64  adrv9009_gpio_14_b     LVCMOS18     #N/A
+A17       GPIO_15_B             AW20        IO_L4P_T0U_N6_DBC_AD7P_64   adrv9009_gpio_15_b     LVCMOS18     #N/A
+A18       GPIO_16_B             AM21        IO_L8N_T1L_N3_AD5N_64       adrv9009_gpio_16_b     LVCMOS18     #N/A
+A19       GPIO_17_B             AL21        IO_L8P_T1L_N2_AD5P_64       adrv9009_gpio_17_b     LVCMOS18     #N/A
+A1        GPIO_18_B             AL10        IO_L23N_T3U_N9_66           adrv9009_gpio_18_b     LVCMOS18     #N/A
+
+B21       GPIO_0_A              AW7         IO_L8P_T1L_N2_AD5P_66       adrv9009_gpio_00_a     LVCMOS18     #N/A
+B22       GPIO_1_A              AW6         IO_L8N_T1L_N3_AD5N_66       adrv9009_gpio_01_a     LVCMOS18     #N/A
+B23       GPIO_2_A              AU5         IO_L9P_T1L_N4_AD12P_66      adrv9009_gpio_02_a     LVCMOS18     #N/A
+B24       GPIO_3_A              AU4         IO_L9N_T1L_N5_AD12N_66      adrv9009_gpio_03_a     LVCMOS18     #N/A
+B25       GPIO_4_A              AV4         IO_L10P_T1U_N6_QBC_AD4P_66  adrv9009_gpio_04_a     LVCMOS18     #N/A
+B26       GPIO_5_A              AW4         IO_L10N_T1U_N7_QBC_AD4N_66  adrv9009_gpio_05_a     LVCMOS18     #N/A
+B27       GPIO_6_A              AT8         IO_L13P_T2L_N0_GC_QBC_66    adrv9009_gpio_06_a     LVCMOS18     #N/A
+B28       GPIO_7_A              AT7         IO_L13N_T2L_N1_GC_QBC_66    adrv9009_gpio_07_a     LVCMOS18     #N/A
+B29       GPIO_8_A              AT5         IO_L14N_T2L_N3_GC_66        adrv9009_gpio_08_a     LVCMOS18     #N/A
+B30       GPIO_9_A              AU9         IO_L15P_T2L_N4_AD11P_66     adrv9009_gpio_09_a     LVCMOS18     #N/A
+B31       GPIO_10_A             AU8         IO_L15N_T2L_N5_AD11N_66     adrv9009_gpio_10_a     LVCMOS18     #N/A
+B32       GPIO_11_A             AR5         IO_L16P_T2U_N6_QBC_AD3P_66  adrv9009_gpio_11_a     LVCMOS18     #N/A
+B33       GPIO_12_A             AR4         IO_L16N_T2U_N7_QBC_AD3N_66  adrv9009_gpio_12_a     LVCMOS18     #N/A
+B34       GPIO_13_A             AR7         IO_L17N_T2U_N9_AD10N_66     adrv9009_gpio_13_a     LVCMOS18     #N/A
+B35       GPIO_14_A             AP7         IO_L17P_T2U_N8_AD10P_66     adrv9009_gpio_14_a     LVCMOS18     #N/A
+B36       GPIO_15_A             AT6         IO_L14P_T2L_N2_GC_66        adrv9009_gpio_15_a     LVCMOS18     #N/A
+B37       GPIO_16_A             AP4         IO_L18N_T2U_N11_AD2N_66     adrv9009_gpio_16_a     LVCMOS18     #N/A
+B38       GPIO_17_A             AP5         IO_L18P_T2U_N10_AD2P_66     adrv9009_gpio_17_a     LVCMOS18     #N/A
+B40       GPIO_18_A             AP6         IO_L18P_T2U_N10_AD2P_66     adrv9009_gpio_18_a     LVCMOS18     #N/A
```

Comment:
For line 43, for `GPIO_18_A` I think the `FPGA_port_name` is mistaken; shouldn't it be `IO_T2U_N12_66`? As per the [schematic](https://wiki.analog.com/_media/resources/eval/user-guides/adrv9009_zu11eg/02-048949-01-c.pdf), page 14 > U21 block.